src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/GetAllocationResourceIntTest.kt

```diff
@@ -235,6 +237,11 @@ class GetAllocationResourceIntTest : IntegrationTestBase() {
               courseAllocation(courseActivity = courseActivity, startDate = "$today")
             }
           }
+          offender {
+            booking {
+              courseAllocation(courseActivity = courseActivity, startDate = "$today", suspended = true)
+            }
+          }
```

Comment:
So all tests in this class now have a suspended prisoner added to the test data and a test on the new suspended endpoint.